src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1593,14 +1592,15 @@ impl Cage {
                     return syscall_error(Errno::EOPNOTSUPP, "rename", "Cannot move file to another directory");
                 }
                 
-                if let Inode::Dir(parent_dir) = metadata.inodetable.get_mut(&parent_inodenum).unwrap() {
+                let pardir_inodeobj = metadata.inodetable.get_mut(&parent_inodenum).unwrap();
+                if let Inode::Dir(parent_dir) = pardir_inodeobj {
```

Comment:
I think this is probably left over from some previous architecture you had, this change doesn't really matter one way or another